src/Controls/tests/TestCases.HostApp/Issues/Issue3643.xaml.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+
+namespace Maui.Controls.Sample.Issues
+{
+
+	[Issue(IssueTracker.Github, 3643, "CollectionView items don't change their size when item content size is changed", PlatformAffected.iOS)]
+	public partial class Issue3643 : ContentPage
```

Comment:
Nice tests, not in this scope of this PR, but would be a good idea to create another test based on this one, doing the same tests but with all the Views.

Reply:
Yea, I'd like to look through all the CV tests that we're porting over to see if there's a set that cover this